### PR TITLE
support language switch for login, register, landing pages

### PR DESCRIPTION
**Pull request by tracytrle**

### CODE REVIEW

File: src/i18n.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import { initReactI18next } from "react-i18next";
+import HttpApi from "i18next-http-backend";
+import LanguageDetector from "i18next-browser-languagedetector";
+
+i18n
+  .use(HttpApi)
+  .use(LanguageDetector)
+  .use(initReactI18next)
+  .init({
+    fallbackLng: "en",
```
Comment (travistrle) [inline src/i18n.js:11]: fall back language should be `vi` since our site is mainly for vietnamese

File: src/layouts/DefaultAppBar.js [hunk 2]
```diff
@@ -55,11 +49,17 @@ const whiteTheme = createTheme({
...
+  const changeLanguage = (lng) => {
+    i18n.changeLanguage(lng);
+  };
+
   return (
     <Stack spacing={2} sx={{ flexGrow: 1 }}>
       <ThemeProvider theme={whiteTheme}>
         <AppBar position="static" color="primary" enableColorOnDark>
-          {appBarLabel("MedicAI")}
+          <AppBarLabel label="MedicAI" changeLanguage={changeLanguage} />
```
Comment (travistrle) [inline src/layouts/DefaultAppBar.js:62]: Thanks for fixing it

File: src/i18n.js (hunk 1 above)
Comment (tracytrle) [reply src/i18n.js:11]: done

File: src/layouts/DefaultAppBar.js (hunk 2 above)
Comment (tracytrle) [reply src/layouts/DefaultAppBar.js:62]: Kho tinh

**Pull request merged**